docs/design/authentication-patterns.md

```diff
@@ -38,8 +33,12 @@ If single sign-on is not yet available to your users, consider an alternative au
 > [!NOTE] 
 > When using Microsoft's Identity service you'll have the opportunity to use a branded sign-in button that is customizable to light and dark themes. Learn more.
 
-## Single Sign-On authentication flow
-Single sign-on is still in preview. Once generally available, use it for the smoothest end-user experience. The user's identity within Office is used to sign-in to your add-in. As a result users only sign-in once. This removes friction in the experience making it easier for your customers to get started.
+## Single Sign-On authentication flow (preview)
+
+> [!NOTE]
+> The single sign-on API is currently supported in preview for Word, Excel, Outlook, and PowerPoint. For more information about where the single sign-on API is currently supported, see [IdentityAPI requirement sets](https://docs.microsoft.com/office/dev/add-ins/reference/requirement-sets/identity-api-requirement-sets?view=office-js). If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Office 365 tenancy. For information about how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
+
+Once single sign-on is generally available for production add-ins, use it for the smoothest end-user experience. The user's identity within Office is used to sign-in to your add-in. As a result users only sign-in once. This removes friction in the experience making it easier for your customers to get started.
```

Comment:
Also, this might be out of the scope of this PR, but it's unclear to me what the user's "identity within Office" is.  Is that my Office/Microsoft account?